src/Dfc.CourseDirectory.WebV2/Features/Venues/DeleteVenue/DeleteVenue.cshtml

```diff
@@ -0,0 +1,183 @@
+
+            <h1 class="govuk-heading-xl">Delete location</h1>
+
+            <govuk-summary-list>
+                <govuk-summary-list-row>
+                    <govuk-summary-list-row-key>Location name</govuk-summary-list-row-key>
+                    <govuk-summary-list-row-value>@Model.VenueName</govuk-summary-list-row-value>
+                </govuk-summary-list-row>
+                <govuk-summary-list-row>
+                    <govuk-summary-list-row-key>Address</govuk-summary-list-row-key>
+                    <govuk-summary-list-row-value>
+                        @foreach (var part in Model.AddressParts)
+                        {
+                            @part<br />
+                        }
+                    </govuk-summary-list-row-value>
+                </govuk-summary-list-row>
+                <govuk-summary-list-row>
+                    <govuk-summary-list-row-key>Postcode</govuk-summary-list-row-key>
+                    <govuk-summary-list-row-value>@Model.PostCode</govuk-summary-list-row-value>
+                </govuk-summary-list-row>
+            </govuk-summary-list>
+
+            @if (Model.AffectedCourses.Count > 0
+               || Model.AffectedApprenticeships.Count > 0
+               || Model.AffectedTLevels.Count > 0)
+            {
+                <govuk-warning-text icon-fallback-text="Warning">
+                    You cannot delete this location
+                </govuk-warning-text>
+
+                <p class="govuk-body">You will need to change the location or delete the following before this location can be deleted.</p>
+
+                @if (Model.AffectedCourses.Count > 0)
+                {
+                    <div class="govuk-form-group@(affectedCoursesErrors.Any() ? " govuk-form-group--error" : string.Empty)">
+
+                        @foreach (var error in affectedCoursesErrors)
+                        {
+                            <govuk-error-message>
+                                @error.ErrorMessage
+                            </govuk-error-message>
+                        }
+
+                        <govuk-details>
+                            <govuk-details-summary>@Model.AffectedCourses.Count Course@(Pluralise(Model.AffectedCourses.Count > 1))</govuk-details-summary>
+                            <govuk-details-text>
+                                <p class="govuk-body">The following courses are linked to this location</p>
+                                <ul class="govuk-list govuk-list--bullet">
+                                    @foreach (var affectedCourseRun in Model.AffectedCourses.OrderBy(c => c.CourseName))
+                                    {
+                                        <li data-testId="affected-course-@affectedCourseRun.CourseId"><a asp-controller="CourseSummary" asp-action="Index" asp-route-courseId="@affectedCourseRun.CourseId" asp-route-courseRunId="@affectedCourseRun.CourseRunId" class="govuk-link">@affectedCourseRun.CourseName</a></li>
```

Comment:
wrap because github reivew interface can't, also a little hard to mentally parse like this. suggest breaking before attributes to avoid introducing whitespace to output

Reply:
```suggestion
                                        <li data-testId="affected-course-@affectedCourseRun.CourseId"><a
                                        asp-controller="CourseSummary" asp-action="Index" asp-route-courseId="@affectedCourseRun.CourseId" asp-route-courseRunId="@affectedCourseRun.CourseRunId"
                                          class="govuk-link">@affectedCourseRun.CourseName</a></li>
```
or similar, same for other really long lines in this file

Reply:
I've wrapped it as sensibly as HTML can be wrapped.